introduction/Java_Currency_Formatter/Solution.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Locale;
+import java.util.Locale.Builder;
```

Comment:
Don't import `Builder`s, use `Locale.Builder` in code, it will be clearer what is being built.